geoengine/permissions.py

```diff
@@ -84,7 +84,7 @@ def __repr__(self) -> str:
 class ResourceId:
     '''A wrapper for a resource id'''
 
-    def __init__(self, resource_type: Literal['DatasetId', 'Layer', 'LayerCollection'],
+    def __init__(self, resource_type: Literal['DatasetName', 'Layer', 'LayerCollection'],
```

Comment:
I think it's still called `DatasetId` in the backend

Reply:
I've changed it in the backend.

[81b1983](https://github.com/geo-engine/geoengine-python/pull/136/commits/81b1983e092036eb315c9628f2ca3f7e79e3fd66)

Reply:
I tried to test it by running the add_public_raster_dataset.ipynb Notebook but I get an error on `get_xarray`:

> ServiceException: {"error":"UnableToParseQueryString","message":"Unable to parse query string: Invalid time 2014-04-01T12:00:00.000 00:00"}

Reply:
Remove your dependencies and reinstall the library. This has something to do with the library changes @jdroenner  did back then.